src/loaders/MaterialLoader.js

```diff
@@ -147,6 +147,20 @@ MaterialLoader.prototype = Object.assign( Object.create( Loader.prototype ), {
 
 		}
 
+		if ( json.vertexAlpha !== undefined ) {
+
+			if ( typeof json.vertexAlpha === 'number' ) {
```

Comment:
> if ( typeof json.vertexAlpha === 'number' ) {

Did you copy this code from `vertexColors`? Notice that it does not make sense to use it in context of `vertexAlpha`. The if statement comes from a time where `vertexColors` was not a boolean but an enum (with values like `THREE.VertexColors`, `THREE.FaceColors`). It was later converted to a boolean. The if statement exists only for backwards compatibility reason.

Reply:
Yes I did a copy. Didn't really understand the legacy code. Will change it.